vue-custom-bootstrap/src/components/CheckboxInput.vue

```diff
@@ -4,16 +4,20 @@
     :invalid-feedback="errorText"
     :state="hasError()"
     :label-for="id"
```

Comment:
I think you can remove label-for from here since there's no label anymore.

Reply:
okay